anxcloud/setup_test.go

```diff
@@ -0,0 +1,30 @@
+package anxcloud
+
+import (
+	"github.com/anexia-it/terraform-provider-anxcloud/anxcloud/testutils/environment"
+	"log"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+)
+
+func TestMain(m *testing.M) {
+	rand.Seed(time.Now().Unix())
```

Comment:
maybe use UnixNano in case we ever decide to run tests in parallel for e.g. different locations, in which case them starting in the same second is possible

Reply:
ack